integration/test.ats

```diff
@@ -68,8 +72,13 @@ def tolerance_test(name, input_file, num_mpi_tasks=1, tolerance=0.00001):
         Path to the input file
     num_mpi_tasks : int, optional
         The number of MPI tasks (default is 1)
-    tolerance : int, optional
+    tolerance : str, optional
         The allowed tolerance in test results (default is 0.00001)
+        Three options:
+            1.) Either single value such as 0.00001
+            2.) Multiple values in this format: default:0.1,velocity:0.22
+            3.) A JSON file containing tolerances. (Located in tests/integration/tolerances)
+                Only the JSON file name is required (Example: tolerances_default.json)
```

Comment:
This shouldn't be limited to `tests/integration/tolerances`.  I can see individual tests needing their own tolerance that lives in their directory.  I would have it just be `tests/integration` instead or add some logic to check the tests directory.

Also `tolerances_default.json` does not exist anymore and this should be renamed to `tolerance_example.json`.

Reply:
Right this is a mistake by me. You need to specify the full path at the moment.

I feel like we should still have a `tests/integration/tolerances` as an optional location, but I agree with simply preceding the tolerance location with `serac_repo/tests/integration` to let users put the `.json` files where they want.

Should I add the "`serac_repo/tests/integration`" stuff to the tolerance file inside `tolerance_test.py` or `tests.ats`? I think `test.ats` would be easier to implement.

Reply:
`test.ats` is probably fine. Up to you if you want to do it in this PR or a follow-up...

Reply:
I had to define `serac_integration_test_dir` and then I was able to get something like the code example above to work. @white238 this should be ready for final review.